src/gdk2.nim

```diff
@@ -1035,6 +1035,17 @@ proc get_clip_region*(drawable: PDrawable): PRegion{.cdecl,
     dynlib: lib, importc: "gdk_drawable_get_clip_region".}
 proc get_visible_region*(drawable: PDrawable): PRegion{.cdecl,
     dynlib: lib, importc: "gdk_drawable_get_visible_region".}
+proc text*(drawable: PDrawable, font: PFont, gc: PGC, x: gint,
```

Comment:
I understand what you mean. But then it would be inconsistent to all other procs like line() -> gdk_draw_line(), rectangle() -> gdk_draw_rectangle(), ...